### PR TITLE
South Africa -- Ntshembo Hlongwane

### PR DESCRIPTION
**Pull request by Ntshembo-Hlongwane1**
# Your Details

- Your Name: Ntshembo
- Your City: Cape Town
- Your Slack Name: Ntshembo Hlongwane

# Homework Details

- Module: Git & Github 
- Week: 1

### CODE REVIEW

**Comment by RynoFourieZA** [comment]
Good Day Ntshembo, hope everything is good and well. I want to point out on questions 4 and 12 you have a typo error.

**Comment by Ntshembo-Hlongwane1** [comment]
Hey Ryno great catch! However I do not see a typo in question 12 could you at least highlight specifically where you see that.

Q12  --> What is the Git History log?
>  This is the entire log/list of all your commits that you have made in that repo/branch if the is more than one branch

**Comment by RynoFourieZA** [comment]
>``` -This is the entire log/list of all your commits that you have made in that repo/branch if the is more than one branch
+ - This is the entire log/list of all your commits that you have made in that repo/branch if there is more than one branch
```

**Comment by RynoFourieZA** [comment]
Good day, Ntshembo sorry for the late reply, as I'm reading through question 12 there seems to be no typo error I thought you needed to replace the second _the_ with _there_.

File: homework.md [hunk 1]
```diff
@@ -4,110 +4,111 @@
 
 What is 2 + 2?
 
-5
+- 4
```
Comment (Samuelkau) [inline homework.md:7]: Your answer looks correct

File: homework.md [hunk 2]
```diff
@@ -4,110 +4,111 @@
...
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+- White
```
Comment (Samuelkau) [inline homework.md:13]: Light

**Review by Samuelkau — CHANGES_REQUESTED** [review]

**Comment by KudahShambare** [comment]
Hello bro, consider filling your details on the first comment.

**Comment by Ntshembo-Hlongwane1** [comment]
@KudahShambare What details please highlight that I'm lost

**Comment by KudahShambare** [comment]
Your Name:
Your City:
Your Slack Name:
........This template, I think that you are supposed to fill it up.

**Comment by Ntshembo-Hlongwane1** [comment]
Oh that I thought that was not necessary but okay let me do it

**Pull request closed without merging**